content/protecting-data-against-ai.en.md

```diff
@@ -0,0 +1,126 @@
+
+## FAQs
+
+**Is Cryptomator compatible with my cloud service?**
+
+Yes, Cryptomator supports most popular cloud services such as Google Drive, Dropbox, OneDrive and many more. The app creates a virtual drive on your device to which you move your files. These files are then encrypted and stored in your cloud storage.
+
+**Is Cryptomator also suitable for mobile use?**
+
+Yes, Cryptomator offers apps for iOS and Android. These mobile apps allow you to securely access your encrypted data on the go. They are intuitive to use and offer the same security standards as the desktop version.
+
+**Can I use Cryptomator for free?**
+
+Yes, Cryptomator offers a free version with all the basic functions. There is also a [paid version](https://cryptomator.org/supporter-cert/) with additional features such as dark mode and support for automatically locking vaults after a certain period of time.
+
+**How does Cryptomator differ from other encryption solutions?**
+
+Cryptomator is specifically designed for the encryption of cloud data and offers seamless integration with common cloud services. Unlike many other solutions, Cryptomator is an open source application, which means that the [source code is publicly available](https://github.com/cryptomator) and verifiable. This increases confidence in the security and integrity of the software. Cryptomator is also easy to use and requires no prior technical knowledge.
+
+**Can I integrate Cryptomator into my existing backup strategy?**
+
+Yes, Cryptomator can be easily integrated into existing backup strategies. Since the app creates a virtual drive on your device, you can integrate your encrypted data into your backup program as usual. This ensures that both the data backup and the encryption of your files are guaranteed.
+
+**How secure is Cryptomator's encryption?**
+
+Cryptomator uses AES-256 bit encryption, which is considered very secure. AES-256 is a symmetric encryption method that is recognized and used by security authorities and companies worldwide. Each file is encrypted individually, which further increases data security. In addition, file name obfuscation ensures that the names of your files and folders are also protected.
+
+**How does file name obfuscation work?**
+
+File name obfuscation is a function that ensures that not only the content but also the names of your files and folders are encrypted. This prevents anyone without the appropriate authorizations from being able to draw conclusions about the content of your files. Cryptomator uses a special algorithm that converts the file names into a random character string.
+
+**What do I do if I forget my password?**
+
+The password you set for your Cryptomator vault is crucial for accessing your encrypted data. If you forget it, there is no way to reset it as Cryptomator does not use a central server or backdoor for password access. It is therefore important to keep your password safe and, if necessary, save it in a password manager.
```

Comment:
We need to mention the recovery key here https://docs.cryptomator.org/en/latest/desktop/password-and-recovery-key/

Reply:
The passage was adjusted in commit da4430a.